packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-sign-in-pwd-phone/main/go/integrationsteps.md

```diff
@@ -96,12 +106,22 @@ return
 The next step is to build a page that allows the user to choose a factor
 to continue the authentication flow.
 
-<div class="half">
+<div class="half wireframe-border">
 
-![Displays an option to choose verification through the phone factor](/img/oie-embedded-sdk/oie-embedded-sdk-go-use-case-phone-verify-page.png)
+![Choose authenticator form with phone option](/img/oie-embedded-sdk/wireframes/choose-authenticator-phone-form-g3r18.png)
 
 </div>
 
+<!--
+
+Source image:
+
+https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%8C%9F-Updated-Diagrams-for-Dev-Docs?node-id=2393%3A2128#233281241
+
+Group 3, row 18
+
+-->
+
```

Comment:
On line 177, just above "7: Submit the phone factor details", you've missed replacing a screenshot for a wireframe.

Reply:
Ah, this is the other example of where one platform doesn't follow the same flow/app design as the others, therefore I wasn't sure that the wireframes were suitable in this case, so I just left it. What do you think?

Reply:
This is fine just to have the request for phone number wireframe image here as long as you adjust the sentence immediately preceding it. /img/oie-embedded-sdk/wireframes/auth-enter-phone-number-form-g2r32.png

Reply:
Yup that's fien @chrisdmills-okta Approved!